### ISSUE TITLE
Timeline: Insert/Delete Layer buttons

### ISSUE THREAD

**Issue by gab3d** [opentoonz/opentoonz#3996]
I really feel the current Timeline UX is confusing, regarding simple layer management.

We have unused layers hidden by default, which is good I think.
But then, that triggers the need to be able to simply add/remove layers as you go. And that is not handled in a good way currently, I'm affraid.

The most prominent button, users will land their eyes on while looking for how to add a new layer is the "Add New Memo" button, which isn't a very frequent need, but then there's no buttons for the much more common Add/Remove layer operations.
So I think we could improve the experience by showing a couple of buttons [ + /  - ] somewhere (most probably on the same row the current Visibility and Lock buttons are).

I know those buttons would probably just be visual ways to access the Insert and Delete column commands, but having them visible all the time could be beneficial also by not having to have a layer header selected to use them (instead they'd just insert or delete the currently selected cell layer).
What do you think?

**Comment by beeheemooth** [comment]
The reason I still don't use the timeline is because I have to keep creating those damn empty layers all the time. Also, the copy-paste of frames of several layers in the timeline is not the same as in the Xsheet. Xsheet has total freedom of action, no need to create empty layers, and in the timeline I constantly stumble and feel a feeling reminiscent of claustrophobia, hehe.

**Comment by artisteacher** [comment]
I think that would be very helpful.

**Comment by RodneyBaker** [comment]
A good start would be to add one additional (empty) row and each time that row becomes populated have the Timeline add a new empty row.  While this would still not make the Timeline as versatile as the xsheet for drawing in a layered workflow it would save a whole lot of work creating new rows when using the Timeline.

**Comment by gab3d** [comment]
yes, always having an empty layer visible at the top row would be a positive thing too.

**Comment by age3rcm** [comment]
only for timeline sheet right?
i think it would break the vertical xsheet to have this button

**Comment by gab3d** [comment]
@age3rcm 
yes
